web/stories/ui/pagination-bar.stories.svelte

```diff
@@ -0,0 +1,153 @@
+<script context="module" lang="ts">
+  import PaginationBar from '../../ui/pagination-bar.svelte'
+
+  export const meta = {
+    component: PaginationBar,
+    title: 'UI/PaginationBar'
+  }
+</script>
+
+<script lang="ts">
+  import { Story } from '@storybook/addon-svelte-csf'
+
+  import Card from '../../ui/card.svelte'
+  import Paragraph from '../../ui/paragraph.svelte'
+  import Section from '../section.svelte'
+
+  let currentPage = 1
+  let postsPerPage = 50
+  let totalPosts = 628
+
+  $: totalPages = Math.ceil(totalPosts / postsPerPage)
+  $: postsFrom = (currentPage - 1) * postsPerPage + 1
+  $: postsTo = Math.min(currentPage * postsPerPage, totalPosts)
+</script>
+
+<Story name="Base">
```

Comment:
We need more cases for Storybook.

We use it not for manually check, but for automatic snapshots tests. So we need at least tests for:
1. We have only 1 page (no pagination)
2. We have 2 pages
3. We have 3 pages
4. We have 3 pages and the current one is 2
5. We have 3 pages and the current one is the last
6. We have 500 pages

We need all these tests only for the light variant